packages/esbuild/README.md

```diff
@@ -0,0 +1,12 @@
+# Size Limit Esbuild Plugin
+
+The plugin for [Size Limit] to track dependencies size by Esbuild.
```

Comment:
We need to fix `Esbuild` → `esbuild` across all docs, not only in the place where I showed

Reply:
Changed, I've set `"(by |)esbuild",` for the yaspeller dictionary. How can it be properly handled?